test/python/circuit/test_circuit_load_from_qpy.py

```diff
@@ -2250,6 +2250,22 @@ def test_pre_v14_rejects_stretch_expr(self, version):
         ):
             dump(qc, fptr, version=version)
 
+    @ddt.idata(range(14, 16))
+    def test_pre_v16_rejects_ps_duration(self, version):
```

Comment:
Is there a test that validates we can roundtrip a picosecond unit delay in version 16?

Reply:
Good point. I added it to our `test_qpy_roundtrip_control_flow_expr` test circuit. Which it actually turns out was completely dead code (looks like a copy-paste mistake). It's wired up now, and luckily everything works 😅.